client/app/vps/import/vps.service.js

```diff
@@ -688,7 +688,7 @@ angular.module("managerApp").service("VpsService", [
         };
 
         this.getOptionSnapshotFormated = function (serviceName, vps) {
-            if (vps.version === "_2015_V_1" && vps.offerType === "SSD") {
+            if ((vps.version === "_2015_V_1" || vps.version === "_2018_V_1") && vps.offerType === "SSD") {
```

Comment:
Otherwise you could write this:

```js
_.includes(["_2015_V_1", "_2018_V_1"], vps.version)
```

Reply:
Just modified to use includes, thank you very much.